README.rst

```diff
@@ -134,6 +134,31 @@ Basic usage
     a_view.an_input.is_displayed
 
 
+``__locator__()`` and ``__element__()`` protocol
+------------------------------------------------
+
+To ensure good structure, a protocol of two methods was introduced. Let's talk a bit about them.
+
+``__locator__()`` method is not implemented by default on ``Widget`` class. Its sole purpose is to
```

Comment:
Only if you pass it ROOT, then it is generated in the meta class. Otherwise you can have a widget that does not have a `__locator__`